level2/user-data.sh

```diff
@@ -0,0 +1,7 @@
+#!bin/bash
+yum update -y
+yum install httpd git -y
+git clone https://github.com/gabrielecirulli/2048.git
+cp -R 2048/* /var/www/html
+systemctl start httpd
+systemctl enable httpd
```

Comment:
could you please, add a newline at the end of file.

Reply:
done.